src/Accounts/ChartValidationSettings.js

```diff
@@ -8,6 +8,7 @@ class ChartValidationSettings {
         this.VALIDATE_UNLABELED_OBJECTS = chartValidations.VALIDATE_UNLABELED_OBJECTS
         this.VALIDATE_FOCAL_POINT = chartValidations.VALIDATE_FOCAL_POINT
         this.VALIDATE_OBJECT_TYPES_PER_CATEGORY = chartValidations.VALIDATE_OBJECT_TYPES_PER_CATEGORY
+        this.VALIDATE_EMPTY_FLOOR = chartValidations.VALIDATE_EMPTY_FLOOR
```

Comment:
Looks like we check for the presence of these keys in `retrieveMyAccount.test.js`

Reply:
Added!

Reply:
Keep in mind that the test will fail because staging is not updated

Reply:
👍